src/main/java/org/jitsi/jicofo/BridgeSelector.java

```diff
@@ -593,7 +659,19 @@ synchronized public Version getBridgeVersion(String bridgeJid)
          * If not set we consider it highly occupied,
          * because no stats we have been fetched so far.
          */
-        private int videoStreamCount = Integer.MAX_VALUE;
+        private Integer videoStreamCount = null;
```

Comment:
Have you considered making this a primitive and equal to 0 initially? (I'm not suggesting that we should make it a primitive, just asking)